muhasin/index.js

```diff
@@ -0,0 +1,23 @@
+const euroCities = ["Paris", "London", "Valletta", "Prague", "Rome"];
+
```

Comment:
After line 1 you have to store the second item of euroCities array in another variable